strutil/strutil.go

```diff
@@ -127,3 +127,23 @@ func SortedListContains(list []string, str string) bool {
 	}
 	return list[i] == str
 }
+
+// TruncateOutput truncates input data by maxLines, imposing maxBytes limit (total) for them.
+// The maxLines may be 0 to avoid the constraint on number of lines.
+func TruncateOutput(data []byte, maxLines, maxBytes int) []byte {
+	if maxBytes > len(data) {
+		maxBytes = len(data)
+	}
+	lines := maxLines
```

Comment:
```
if maxLines == 0 && len(data) <= maxBytes {
    return data
}
```

Reply:
Thanks, good idea. I actually expanded it for the simple case when lines == 0, and we can immediately return a slice (or all data).